server/services/drafts/draftReportService.ts

```diff
@@ -119,6 +123,32 @@ export default class DraftReportService {
     return this.handleAddStaff(token, user, bookingId, [userToAdd])
   }
 
+  async getPotentialDuplicates(
+    bookingId: number,
+    incidentDate: Moment,
+    token: string
+  ): Promise<Partial<DuplicateReport[] | []>> {
+    const startDate = moment(incidentDate).startOf('day')
+    const endDate = moment(incidentDate).endOf('day')
+    const reports = await this.draftReportClient.getDuplicateReports(bookingId, [startDate, endDate])
+    return Promise.all(
+      reports
+        .filter(report => report.status !== ReportStatus.IN_PROGRESS.value)
+        .map(async r => {
+          const location = await this.locationService.getLocation(token, r.form.incidentDetails.locationId)
+          return {
+            reporter: r.reporter,
+            date: moment(r.date),
+            location: location?.userDescription?.toString(),
+          }
```

Comment:
question mark after location was required

Reply:
this is presumably just an issue with the tests - the location will always be present.. not a big problem though

Reply:
yes, it was causing tests to fail